.github/workflows/rebuildDependencies.yml

```diff
@@ -62,12 +69,27 @@ jobs:
         repository: 'vcmi/vcmi'
         ref: 'update_prebuilts'
 
-    - name: Install dependencies
+    - name: Prepare CI
       run: source '${{github.workspace}}/CI/${{matrix.platform}}/before_install.sh'
       env:
         VCMI_BUILD_PLATFORM: x64
 
-    - name: Remove old packages
+    - name: Install Conan Dependencies
+      if: "${{ matrix.conan_prebuilts != '' }}"
+      run: source '${{github.workspace}}/CI/install_conan_dependencies.sh' '${{matrix.conan_prebuilts}}'
+
+    - name: Remove old binary packages
+      if: ${{ !startsWith(matrix.platform, 'android') }}
+      run: rm -rf ~/.conan/data/*/*/_/_/package
+    
+    - name: Remove old binary packages
+      if: ${{ startsWith(matrix.platform, 'android') }}
+      run: |
+        mv ~/.conan/data/libiconv ~/
+        rm -rf ~/.conan/data/*/*/_/_/package
+        mv ~/libiconv ~/.conan/data
```

Comment:
what's so special about libiconv for android?

Reply:
Fails to build with no clear reason: 
https://github.com/vcmi/vcmi-dependencies/actions/runs/10492872885/job/29065442388
Any ideas what could be the reason?

At the moment trying to figure out which packages fail to rebuild from source.
For Android - qt also fail to build: https://github.com/vcmi/vcmi-dependencies/actions/runs/10509979837/job/29117474852
Although I think this is simply because workflow does not have java installed right now?

Newer mingw fails to build with some weird conan error, but older one that we use currently is fine:
https://github.com/vcmi/vcmi-dependencies/actions/runs/10509979837/job/29117473538

Rest libraries / platforms seems fine so far.

Reply:
yeah, unclear what's wrong, but seems that .so isn't created:
```
# no .so here
libtool: link: ( cd ".libs" && rm -f "libcharset.la" && ln -s "../libcharset.la" "libcharset.la" )
...
libtool: install: /usr/bin/install -c .libs/libcharset.so /Users/runner/.conan/data/libiconv/1.17/_/_/build/1b95afa31f2b18c56461568c63bcead97119b0e6/build-release/lib/libcharset.so
install: .libs/libcharset.so: No such file or directory
```

however, no such issue on ConanCenter. Maybe switching to Linux would fix that.